src/java.base/share/classes/jdk/internal/crac/JDKFileResource.java

```diff
@@ -25,9 +25,14 @@ public abstract class JDKFileResource extends JDKFdResource {
                 .split(File.pathSeparator);
         CLASSPATH_ENTRIES = new Path[items.length];
         for (int i = 0; i < items.length; i++) {
-            // On Windows, path with forward slashes starting with '/' is an accepted classpath
-            // element, even though it might seem as invalid and parsing in Path.of(...) would fail.
-            CLASSPATH_ENTRIES[i] = new File(items[i]).toPath();
+            try {
+                // On Windows, path with forward slashes starting with '/' is an accepted classpath
+                // element, even though it might seem as invalid and parsing in Path.of(...) would fail.
+                CLASSPATH_ENTRIES[i] = new File(items[i]).toPath();
+            } catch (Exception e) {
+                // Ignore any exception parsing the path: URLClassPath.toFileURL() ignores IOExceptions
+                // as well, here we might get InvalidPathException
```

Comment:
I think `CLASSPATH_ENTRIES[i]` will remain `null` in such case and later code in this file accesses it without a `null` check. But I haven't tried to reproduce it on Windows.

Reply:
Correct, I need to push an update to this.

Reply:
Updated, with improved test.